NewSpring/ApollosProject
# [DEV-4169, DEV-4172] Update Links

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR:
* Swaps the join a group and find a serving opportunity links in the action table and action bar.
* Fixes the Bug Report link to point to the right workflow.

<img width="584" alt="Screenshot 2019-10-27 22 46 26" src="https://user-images.githubusercontent.com/3229463/67648283-a2098480-f90b-11e9-9aae-3eddc4000b41.png">

### How do I test this PR?
* Make sure that the "Join Group" link is in the action bar.
* Make sure that the "Find a serving opportunity" link is in the action table.
* Make sure that the Bug Report link points to the right version of the workflow. You can tell this because the page you go to should have a link to the apps page at NewSpring's website and there should NOT be a field for "Page" (the page of the website that you had a problem on). See the screenshot above.
* Test all the other links just to make sure that they are going to the correct places.

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
